README.md

```diff
@@ -1,172 +1,156 @@
 
 ```
-import "gopkg.in/dedis/cothority.v0"
+go get -u github.com/dedis/cothority/cosi
 ```
 
-### Master
+#### Configuration
 
-The master-branch is used for day-to-day development and will break your code about once a week. If you are using this branch, be sure to do
+To tell the CoSi client which existing cothority (public key) it should use for signing requests (signature verification), you need to specify a configuration file. For example, you could use the [DEDIS cothority configuration file](dedis-cothority.toml) which is included in this repository. To have a shortcut for later on, set:
 
 ```
-go get -u -t ./...
+export COTHORITY=$GOPATH/src/github.com/dedis/cothority/dedis-cothority.toml 
 ```
 
-from time to time, as all dedis-dependencies change quite often.
-
-# Installation
+#### Usage
 
-There are three apps available:
+To request a collective (Schnorr) signature `file.sig` on a `file` from the DEDIS cothority, use:
 
-* [cothorityd](https://github.com/dedis/cothority/app/cothorityd) - which is the server-part that you can run to add a node
-* [CoSi](https://github.com/dedis/cothority/app/cosi) - the CoSi-app
-* [status](https://github.com/dedis/cothority/app/status) - reads out the status of a cothority
+```
+cosi sign -g $COTHORITY -o file.sig file
+```
 
-You will find a README.md in each of its directory. To build the apps, you can
-run the following commands:
+To verify a collective (Schnorr) signature `file.sig` of the `file`, use:
 
 ```
-go get github.com/dedis/cothority/app/cothorityd
-go get github.com/dedis/cothority/app/status
+cosi verify -g $COTHORITY -s file.sig file
 ```
 
-# Apps
+### Cothority Server
+
+Conodes are linked together to form cothorities, run decentralized protocols, and offer services to clients.
+
+#### Installation
+
+To build and install the conode binary, execute:
+
+```
+go get -u github.com/dedis/cothority/conode
+```
 
-* [cothorityd](app/cothorityd) - the basic 
-* [cosi](cosi) - collective signatures
-* [status](status) - returns the status of the given group
-* [cisc](cisc) - handle your ssh-keys on a blockchain
-* [hotpets](https://github.com/dedis/cothority/tree/hpets16/app/cisc) - hotpets16-branch
+To get an overview on the functionality of a conode, type:
 
-# Protocols
+```
+conode help
+```
 
-## CoSi - Collective Signing
+#### Configuration
 
-[CoSi](http://dedis.cs.yale.edu/dissent/papers/witness-abs) is a 
-protocol for scalable collective signing, which enables an authority or 
-leader to request that statements be publicly validated and (co-signed) 
-by a decentralized group of witnesses. 
-Each run of the protocol yields a single digital signature with size and 
-verification cost comparable to an individual signature, but compactly
-attests that both the leader and perhaps many witnesses observed and 
-agreed to sign the statement.
+To configure your conode you need to *open two consecutive ports* (e.g., 6879 and 6880) on your machine, then execute
```

Comment:
Maybe a very short explanation of why we need two ports? 
`One port is used to communicate with other conodes, and the second one is used by external client applications wishing to interacts with a conode over websockets`